src/Administration/Resources/app/administration/vite.config.mts

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
this file is another example of possible solution, but that would be a big refactor - I wouldn't go for that now, but would be useful in future

1. Autoimport loads all components,
2. when component is "overriden" then autoimport just points to different file 

thanks to that you can stack components and they are globally registered as well, similar how it works in Nuxt.
But this file is not actual solution for todays problem